spec/support/macros.rb

```diff
@@ -0,0 +1,11 @@
+def set_current_user(user)
+  session[:user_id] = user.id
+end
+
+def sign_in(user)
+  user = user || Fabricate(:user)
+  visit sign_in_path
+  fill_in "Email Address", :with => user.email
+  fill_in "Password", :with => user.password
```

Comment:
I would stay aligned with the Ruby hash style, you are using the old pre-1.9 syntax style.